swe/solver.py

```diff
@@ -106,8 +107,8 @@ def setup_solver(self):
 
         # Initial conditions  # TODO: will this work over mesh iterations?
         if self.prev_solution is not None:
-            interp = self.interpolated_solution
-            u_interp, eta_interp = self.interpolated_solution.split()
+            interp = self.interpolate_solution
+            u_interp, eta_interp = self.interpolate_solution.split()
```

Comment:
This is some outdated code and `self.interpolated_solution` is no longer an attribute. Please replace with
```u_interp, eta_interp = self.solution.split()```